### PR TITLE
Enh/php cs

### PR DESCRIPTION
**Pull request by skjnldsv**
Wait for https://github.com/nextcloud/bookmarks/pull/1139

### CODE REVIEW

**Comment by marcelklehr** [comment]
For some reason this seems to break on nextcloud 18

**Comment by skjnldsv** [comment]
Ready to go @marcelklehr  :)

File: lib/Activity/Provider.php.orig [hunk 1]
```diff
@@ -0,0 +1,229 @@
+<?php
+
+namespace OCA\Bookmarks\Activity;
+
+use OCA\Bookmarks\Db\Folder;
```
Comment (marcelklehr) [inline lib/Activity/Provider.php.orig:5]: ?

Comment (skjnldsv) [reply lib/Activity/Provider.php.orig:5]: that is what I call, a failure :see_no_evil:

Comment (skjnldsv) [reply lib/Activity/Provider.php.orig]: Resolved

File: composer.lock [hunk 2]
```diff
@@ -1927,536 +2116,2247 @@
...
+                {
+                    "url": "https://tidelift.com/funding/github/packagist/symfony/symfony",
+                    "type": "tidelift"
+                }
+            ],
+            "time": "2020-06-06T08:49:21+00:00"
+        },
+        {
+            "name": "symfony/event-dispatcher",
+            "version": "v5.1.3",
```
Comment (marcelklehr) [inline composer.lock:3331]: This seems to clash with nextcloud's dependency of symfony/event-dispatcher

Comment (skjnldsv) [reply composer.lock:3331]: No idea what that is ^^

Comment (marcelklehr) [reply composer.lock:3331]: Nextcloud core [depends on event-dispatcher v4](https://github.com/nextcloud/3rdparty/tree/master/symfony/event-dispatcher) which is incompatible with v5. PHP CS Fixer also supports v4, though, so you can simply add symfony/event-dispatcher@^4.4 to composer.json and it should work.

Comment (skjnldsv) [reply composer.lock:3331]: Apparently not possible
Your requirements could not be resolved to an installable set of packages.

```
  Problem 1
    - Conclusion: don't install symfony/event-dispatcher v4.4.11
    - Conclusion: don't install symfony/event-dispatcher v4.4.10
    - Conclusion: don't install symfony/event-dispatcher v4.4.9
    - Conclusion: don't install symfony/event-dispatcher v4.4.8
    - Conclusion: don't install symfony/event-dispatcher v4.4.7
    - Conclusion: don't install symfony/event-dispatcher v4.4.6
    - Conclusion: don't install symfony/event-dispatcher v4.4.5
    - Conclusion: don't install symfony/event-dispatcher v4.4.4
    - Conclusion: don't install symfony/event-dispatcher v4.4.3
    - Conclusion: don't install symfony/event-dispatcher v4.4.2
    - Conclusion: don't install symfony/event-dispatcher v4.4.1
    - Conclusion: remove symfony/event-dispatcher-contracts v2.1.3
    - Installation request for symfony/event-dispatcher ^4.4 -> satisfiable by symfony/event-dispatcher[v4.4.0, v4.4.1, v4.4.10, v4.4.11, v4.4.2, v4.4.3, v4.4.4, v4.4.5, v4.4.6, v4.4.7, v4.4.8, v4.4.9].
    - Conclusion: don't install symfony/event-dispatcher-contracts v2.1.3
    - symfony/event-dispatcher v4.4.0 requires symfony/event-dispatcher-contracts ^1.1 -> satisfiable by symfony/contracts[v1.1.1, v1.1.2, v1.1.3, v1.1.4, v1.1.5, v1.1.6, v1.1.7, v1.1.8, v1.1.9], symfony/event-dispatcher-contracts[v1.1.1, v1.1.4, v1.1.5, v1.1.7, v1.1.9].
    - Can only install one of: symfony/event-dispatcher-contracts[v1.1.1, v2.1.3].
    - Can only install one of: symfony/event-dispatcher-contracts[v1.1.4, v2.1.3].
    - Can only install one of: symfony/event-dispatcher-contracts[v1.1.5, v2.1.3].
    - Can only install one of: symfony/event-dispatcher-contracts[v1.1.7, v2.1.3].
    - Can only install one of: symfony/event-dispatcher-contracts[v1.1.9, v2.1.3].
    - don't install symfony/contracts v1.1.1|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.2|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.3|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.4|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.5|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.6|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.7|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.8|don't install symfony/event-dispatcher-contracts v2.1.3
    - don't install symfony/contracts v1.1.9|don't install symfony/event-dispatcher-contracts v2.1.3
    - Installation request for symfony/event-dispatcher-contracts (locked at v2.1.3) -> satisfiable by symfony/event-dispatcher-contracts[v2.1.3].

```

Comment (marcelklehr) [reply composer.lock:3331]: You probably need to remove the lock file.

**Comment by skjnldsv** [comment]
> Fix dependency hell

Thanks :wink:

**Pull request merged**